website/content/docs/configuration/server.mdx

```diff
@@ -198,6 +198,17 @@ server {
   cluster again when starting. This flag allows the previous state to be used to
   rejoin the cluster.
 
+- `root_key_gc_interval` `(string: "10m")` - Specifies the interval
+  between [encryption key] metadata garbage collections.
```

Comment:
Adding an empty set of square brackets will help syntax highlighters properly color the links.

```suggestion
  between [encryption key][] metadata garbage collections.
```

Reply:
TIL!